src/modules/navigator/rtl.cpp

```diff
@@ -228,8 +228,9 @@ void RTL::find_RTL_destination()
 		}
 	}
 
-	if (_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING) {
-		_rtl_alt = calculate_return_alt_from_cone_half_angle((float)_param_rtl_cone_half_angle_deg.get());
+	// For a fixed wing, if we are close to target and inside the cone shape, limit the RTL altitude to current altitude (due to vehicle dynamics)
+	if (_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_FIXED_WING) {
```

Comment:
@junwoo091400 I don't get this. Are you disabling the cone feature for multirotors?

Reply:
I had a confusion. The comment for the parameter has been added in the latest commit to address that it's only meant for a multicopter